pkg/crc/constants/constants.go

```diff
@@ -72,6 +72,7 @@ var (
 	GlobalStatePath    = filepath.Join(CrcBaseDir, GlobalStateFile)
 	DefaultBundlePath  = filepath.Join(MachineCacheDir, GetDefaultBundle())
 	bundleEmbedded     = "false"
+	DaemonSocketPath   = filepath.Join(CrcBaseDir, "crc.sock")
```

Comment:
If logging file is defined here, please create a log path for the daemon too

Reply:
Done.

Reply:
since the log file is called `crcd` it makes sense to make this also `crcd.sock`